automations/js/src/count_user_reviewable_prs.js

```diff
@@ -49,39 +49,46 @@ query ($repoOwner: String!, $repo: String!, $cursor: String) {
     '🟥 priority: critical',
   ]
   const [owner, repo] = GITHUB_REPOSITORY.split('/')
+  const isRelevantPRFromGraphQL = (pr) =>
```

Comment:
Nit:

```suggestion
  const isRelevantPrFromGraphql = (pr) =>
```

I don't want to block a PR for a variable name, but I also want to avoid creating something similar to `XMLHttpRequest`.

As for GraphQL, the most common identifier I've seen for it is 'graphql' (all lowercase), so just uppercasing the 'G' for the camelCase version makes sense to me.

Reply:
Updated!